tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -528,6 +528,45 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
        |""".stripMargin,
   )
 
+  check(
+    "extends-with-class".tag(IgnoreScala2),
```

Comment:
This should also work in Scala 2. Is it not working because of your change or is it working in Scala 3 because of your change? In the latter case let's create an issue for Scala 2 here.

Reply:
I forgot to remove it. It works for both Scala versions.